crates/indexer/src/runtime.rs

```diff
@@ -115,8 +115,13 @@ pub async fn run_forever(config: IndexerConfig, context: Arc<Context>) {
 
     info!(processor_name = processor_name, "Starting indexer...");
 
-    let conn_pool =
-        new_db_pool(&config.postgres_uri.unwrap()).expect("Failed to create connection pool");
+    let db_uri = &config.postgres_uri.unwrap();
+    info!(
+        processor_name = processor_name,
+        database = db_uri,
```

Comment:
this string will have the un/pwd in prod: please don't include it here as you'll leak our credentials into logger